### DISCUSSION TITLE
Need help with finding the intrinsic parameter for AXIS camera

### DISCUSSION THREAD

**Discussion by LiJinAdam** [AxisCommunications/discussions#213] [General]
Dear Axis community, 
      I have an AXIS camera, model number: AXIS P1455-LE Network Camera, and I am trying to find the Camera intrinsic parameter for calibration between the camera and LiDAR. Since this model has a zoom lens, I wonder if the camera can tell the current focal length. In addition, is there a way that the camera can tell its intrinsic parameter? That would be very helpful. Thank you.

**Comment by vivekatoffice** [comment]
Calibration procedure: https://developer.axis.com/acap/develop/camera-calibration/#calibration-procedure
